docs/changelog/3118.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix bug in ``config.rst`` by removing stray colons left over from (:issue:`3111`) - by :user:`posita`. (:issue:`3118`)
```

Comment:
Remove `(:issue:`3118`)` that's generated.

Reply:
https://tox--3120.org.readthedocs.build/en/3120/changelog.html#vnext-2023-09-07 see you're duplicating data

Reply:
Ah! Whoops! I was imitating without understanding. :blush:

![fixed](https://media.tenor.com/bvPdoSdUszMAAAAj/love-heart.gif)